arrow-array/src/array/map_array.rs

```diff
@@ -251,6 +251,69 @@ impl std::fmt::Debug for MapArray {
     }
 }
 
+impl TryFrom<&ListArray> for MapArray {
```

Comment:
```suggestion
impl TryFrom<ListArray> for MapArray {
```
To be consistent with other array conversions, and to make the clone explicit if necessary

Reply:
I removed this one.